include/mgp.hpp

```diff
@@ -2390,6 +2322,27 @@ inline void Node::AddLabel(const std::string_view label) {
   mgp::vertex_add_label(this->ptr_, mgp_label{.name = label.data()});
 }
 
+inline std::map<std::string, Value> Node::Properties() const {
+  mgp_properties_iterator *properties_iterator = mgp::vertex_iter_properties(ptr_, memory);
+  std::map<std::string, Value> property_map_;
+  for (auto *property = mgp::properties_iterator_get(properties_iterator); property;
+       property = mgp::properties_iterator_next(properties_iterator)) {
+    auto value = Value(property->value);
+    property_map_.emplace(std::string(property->name), value);
+  }
+  mgp::properties_iterator_destroy(properties_iterator);
+  return property_map_;
```

Comment:
```suggestion
  std::map<std::string, Value> property_map;
  for (auto *property = mgp::properties_iterator_get(properties_iterator); property;
       property = mgp::properties_iterator_next(properties_iterator)) {
    auto value = Value(property->value);
    property_map.emplace(std::string(property->name), value);
  }
  mgp::properties_iterator_destroy(properties_iterator);
  return property_map;
```

I would also either move `value` into `property_map` or even better: let the compiler do that: `property_map_.emplace(std::string(property->name), Value(property->value));`. This eliminates an unnecessary copy of `Value`.

Reply:
Makese, sense. I will change it and use compailer to move value int map.